doc/source/examples/metrology/Creating Network Definitions for SOLT Calibration Standards.ipynb

```diff
@@ -0,0 +1,590 @@
+    "a termination $Z_0$, its reflection phase shift is assumed to be negligible and is given\n",
+    "a zero offset delay.\n",
+    "\n",
+    "The Thru standard is sometimes modeled with an offset delay only, without loss,\n",
+    "because loss is negligible, especially at low frequencies.\n",
+    "\n",
+    "Conversely, when the Thru is used as a \"flush\" Thru (in most SOLT calibrations,\n",
+    "Thru is first connected to the test cable, and all the Open, Short, Load, and\n",
+    "Thru standards are attached to the other side of the Thru). By definition of the\n",
+    "calibration plane, the Thru standard is completely ideal and zero length, no\n",
+    "modeling is required. In this instance, Keysight still lists an offset loss\n",
+    "for Load and Thru standards, but both are nullified by the zero length."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Preparation\n",
+    "\n",
+    "Equipped with this circuit model, we can now start to model the calibration\n",
+    "standards.\n",
+    "\n",
+    "First, we need to define the frequency range of our calculation and an ideal\n",
+    "coaxial transmission line with air dielectric."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import skrf as rf\n",
+    "from skrf.media import Coaxial\n",
+    "import numpy as np\n",
+    "\n",
+    "freq = rf.Frequency(1, 9000, 1001, \"MHz\")\n",
+    "medium = Coaxial(frequency=freq, z0=50)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Modeling the Offset Transmission Line\n",
+    "\n",
+    "An offset delay is easily created by the `medium.line()` method."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "offset_open  = medium.line(d=29.242, unit='ps')\n",
+    "offset_short = medium.line(d=31.785, unit='ps')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The offset loss is more tricky. We need to model it as a resistor\n",
+    "but it's frequency-dependent and cannot be represented as a single\n",
+    "value. Fortunately, the `medium.resistor()` method conveniently\n",
+    "accepts an array just for this purpose.\n",
+    "\n",
+    "In earlier scikit-rf versions, the method only accepts a `numpy`\n",
+    "array, not a Python list, so a type conversion to `np.array()`\n",
+    "is required."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "resistor_open = []\n",
+    "for f in freq.f:\n",
+    "        r = 2.2 * 29.242 * np.sqrt(f / 1e9)\n",
+    "        resistor_open += [r]\n",
+    "loss_open = medium.resistor(np.array(resistor_open))\n",
+    "\n",
+    "resistor_short = []\n",
+    "for f in freq.f:\n",
+    "        r = 2.36 * 31.785 * np.sqrt(f / 1e9)\n",
```

Comment:
This doesn't look right. The resistance seems way too high and the units here are (Gohms/s) * ps, which doesn't give ohms.

I have understood that the loss is transmission line loss similar to the other definition below. Lossless delay and resistor does not give the same answer as a lossy transmission line. Is this lumped model correct?

Reply:
@Ttl You're right. I rechecked, it cannot be modeled as a lumped resistor. The unit "Ohm" in offset loss comes from the characteristic impedance. I'll rewrite this section.